ivy/functional/ivy/experimental/losses.py

```diff
@@ -421,7 +421,7 @@ def kl_div(
     /,
     *,
     reduction: Optional[str] = "mean",
-    out: Optional[ivy.Array] = None,
```

Comment:
Again, we can't just introduce and remove parameters from the Functional API of ivy without any explaination.

Reply:
Is it necessary for every function to have `out` parameter. I was not aware of that, will make the required changes

Reply:
I'll have a look at `poisson_nll_loss`

Reply:
But yes it is necessary